lib/db.js

```diff
@@ -1,8 +1,9 @@
-const MongoClient = require('mongodb').MongoClient;
-const ObjectID = require('mongodb').ObjectID;
 const assert = require('assert');
-const bcrypt = require('bcrypt')
-const  MongoDB = require('../config/projectInfoData.json')['mongoData']
+const bcrypt = require('bcrypt');
+const ObjectID = require('mongodb').ObjectID;
+const MongoClient = require('mongodb').MongoClient;
+
+const  MongoDB = require('../config/projectInfoData.json')['mongoData'];
```

Comment:
two spaces between cont and MongoDB. Just noticed it and not a huge deal but it is there